decksite/templates/entry.mustache

```diff
@@ -1,5 +1,5 @@
 <li>
-    <a href="{{card.url}}" class="{{^card.pd_legal}}illegal{{/card.pd_legal}} card">{{n}} {{name}}</a>
+    <a href="{{card.url}}" class="{{^card.pd_legal}}illegal{{/card.pd_legal}} card">{{#n}}{{n}} {{/n}}{{name}}</a>
```

Comment:
I would have anticipated you using singlecard for this not entry. But I see that singlecard doesn't have the bug details. I guess these two things are a bit too similar to make sense. I logged #3881 to clean this up.